components/director/examples/merge-applications-with-self-register-distinguish-label-key/merge-applications-with-self-register-distinguish-label-key.graphql

```diff
@@ -0,0 +1,463 @@
+# Code generated by Compass integration tests, DO NOT EDIT.
```

Comment:
This mutation example is a duplicate of the existing `merge-applications` one. Why is it added?

Reply:
It was generated from generate_examples.sh